src/QsFmt/App/Program.fsi

```diff
@@ -7,3 +7,4 @@ module Microsoft.Quantum.QsFmt.App.Program
 /// Runs the Q# formatter.
 [<CompiledName "Main">]
 val main: string [] -> int
+
```

Comment:
Is this newline necessary for correctness?

Reply:
Nope, not sure how that got in there. It is removed now.